packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -705,4 +711,10 @@ class _ResidentWebRunner extends ResidentWebRunner {
     await cleanupAtFinish();
     return 0;
   }
+
+  @override
+  Future<void> exitApp() async {
```

Comment:
Sorry, I just noticed these changes here. Shutting down a web app is a bit complicated - does this change modify the behavior of `flutter run -d chrome` or `flutter run -d chrome --release` when quiting?

Reply:
No, it does not. We got the same behavior.